lib/bedrock.js

```diff
@@ -213,22 +214,22 @@ api.runOnce = function(id, fn, options, callback) {
   options = options || {};
 
   // listen to run function once
```

Comment:
This comment would be in conflict with the change below where we don't always remove the listener once the message has been received. However, please explain why we made that change below before changing this comment -- to ensure it's the right change.